root/statistics/MusicBrainzHistory.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @flow strict
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout/index.js';
+import expand2react from '../static/scripts/common/i18n/expand2react.js';
+
+type PropsT = {
+  +events: Array<StatisticsEventT>,
+};
+
+const MusicBrainzHistory = ({
+  events,
+}: PropsT): React.Element<typeof Layout> => (
+  <Layout fullWidth title={l('History')}>
+    <h1>{l('Our Glorious History')}</h1>
+    {events ? events
```

Comment:
It doesn't look like `events` can be falsey here, judging by the Perl (and existing Flow type). So I think this can lose the ternary.

Reply:
Rather than losing it, it should probably be checking events.length. Will amend.